meta.yaml

```diff
@@ -0,0 +1,42 @@
+
+{% set name = "FlaGs" %}
+{% set version = "1.2.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/chayan7/FlaGs/archive/v1.2.6.tar.gz
+  sha256: 0d64fa8a24ad1d79b9ea570549bf2278cd899ab15f65f63b5b65d7420a229f7d
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install biopython==1.72 ete3 hmmer -vv"
+
+requirements:
+  build:
+  host:
+    - python =3.6
+  run:
+    - python =3.6
+
+
+test:
+  imports:
+    - ete3
```

Comment:
The `imports` tests shouldn't list dependencies. You should attempt to import the module associated with the `FlaGs` package. Of if this isn't a package/library, but is a single file command line tool, then you should test under `commands`:

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-commands

Reply:
Hi I am really struggling with this. So yes my tool needs ete3 , hmmer , biopython 1.72, tkinter to run. I don't know how to make all of them work, suggestion appreciated. Thanks

Reply:
All dependencies should be listed under the requirements section as in:

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#requirements-section

So if biopython is a dependency then you'd need something like:

```
requirements:
  host:
    - python >=3.6
  run:
    - python >=3.6
    - biopython =1.72
    - ete3
    - hmmer
```

but I don't think `hmmer` is currently available on conda-forge so you would need to build a recipe for that. You might also want to think about whether bioconda (https://bioconda.github.io/) would be a more appropriate home for this package.